### ISSUE TITLE
Armoured Stormwind Warhorse - Please redesign this

### ISSUE THREAD

**Issue by Icfre** [slowtorta/turtlewow-bug-tracker#5259]
**Please, provide a clear description what the bug is:**
This mount looks like it was cheaply designed, and its cadence is abnormal compared to all other horse mounts. The animation of it running is jerky, and uneasy. This mount is extremely disappointing in its current state. At the very least, please fix the mount's run animation to look more like a normal horse mount.

**Steps to reproduce the behavior with as much detail as possible:**

1. Get stormwind exalted
2. Farm an all matching transmog set to look identical to a stormwind guard (optional)
3.  Spend 200g on Armour stormwind warhorse
4.  Be disappointed

![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/148420922/eaa935d5-d2c4-4531-87d0-95c293fb2397)

https://imgur.com/a/JrQjNvL (Link to run animation)

**Comment by Icfre** [comment]
Additional issues with this mount that I noticed recently, when you press spacebar while stationary it does not rear (stand on its hind legs) as other horse mounts do. Also, if you jump while mounted its legs stay in a fixed position that also looks dumb.

**Comment by Judo101** [comment]
This model is from beta world of warcraft, therefore, it is missing all the updates nuances of the "current" model horses. I don't believe this horse was ever intended to be a riding mount for players, as it is part of models for certain NPCs and enemies, the first ones that come to mind are the ghostly knights from The Burning Crusade. I personally love mine for its goofiness and I like the extra parts that move in animation (the tassels are my favorite). The only way to redesign it would to rebuild the mount from the ground up on an updated horse model.

**Comment by hypnaceae** [comment]
Correct, this model is technically working as intended. Though I would suggest that the Stormwind Steed from WotLK Argent Tournament would be a much better Stormwind exalted reward than this, which is better suited as a drop from a rare mob, for instance.
